02_knight_and_princess/danil_dyachenko/graph_gen.cpp

```diff
@@ -0,0 +1,122 @@
+#include <array>
+#include <fstream>
+#include <iostream>
+#include <vector>
+
+using VertexId = int;
+using EdgeId = int;
+
+struct Vertex {
+  VertexId id;
+  std::vector<int> edges;
+
+  std::string& to_json() {
+    static std::string string;
```

Comment:
Чего ты пытался тут добиться, расскажешь?
```suggestion
  std::string to_json() {
    std::string string;
```